doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2299,7 +2299,7 @@ Printing universes
    This command can be used to print the constraints on the internal level
    of the occurrences of :math:`\Type` (see :ref:`Sorts`).
 
-   If the optional ``Sorted`` option is given, each universe will be made
+   If the ``Sorted`` option is given, each universe will be made
```

Comment:
What about:\

```suggestion
   If the ``Sorted`` keyword is present, each universe will be made
```

Maybe not a keyword from a parsing perspective though. But option is not the right term here, since we have given it a precise meaning in this manual.

Reply:
"option" is also used to refer to command-line options.  Some overloading of terms is inevitable.

Reply:
Yes, but in this case, I try to insist on using the complete phrasing "command-line option" or "command-line flag" for disambiguation.